docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -83,18 +83,6 @@ deployed.
   result providers, the components that are used by the debugger to parse and
   evaluate C# and VB expressions in the Watch window, Immediate window, and
   more. These components are only used when debugging.
-- **VisualStudioInteractiveWindow**: this project can be found inside the
-  Interactive folder from the Solution Explorer, and builds 
-  Microsoft.VisualStudio.VsInteractiveWindow.vsix.
-  This includes the "base" interactive window experience that is shared by
-  Roslyn, Python, and other languages. This code is core support only and
-  doesn't include any language-specific logic.
-- **VisualStudioSetupInteractive**: this project can be found inside the
-  Interactive\Setup folder from the Solution Explorer, and builds
-  Roslyn.VisualStudio.Setup.Interactive.vsix.
-  It deploys the Roslyn (i.e. C# and VB) specific parts of the interactive
-  window. If you're working on the interactive experience, this is the project
```

Comment:
Should this text be placed somewhere else now about what to F5 for interactive stuff?